.travis.yml

```diff
@@ -4,9 +4,7 @@ node_js:
   - "10"
   - "9"
   - "8"
-  - "7"
   - "6"
-  - "4"
 env:
```

Comment:
If we're dropping support for Node 4 and 7, should we also drop support for some old versions of ESLint? There's currently [some logic](https://github.com/prettier/eslint-plugin-prettier/blob/9fac6b4039c1983b83073fa7af7864f0d7e1f2d3/eslint-plugin-prettier.js#L51-L83) that only runs on versions of ESLint before `3.16.0`.

Reply:
I have no strong preferences either way. I suppose we might as well drop support for 4.x (there aren't many breaking changes in ESLint v5, so hopefully it shouldn't be too difficult for users to upgrade).